File: 1925-count-square-sum-triples/1925-count-square-sum-triples.js
/**
 * @param {number} n
 * @return {number}
 */
var countTriples = function(n) {
    let counter = 0;
    for(let i = n; i >= 1; i--){
        for(let j = n; j>= 1;j--){
            let c = Math.sqrt(i*i + j*j);
            if(Number.isInteger(c) && c <= n){
                counter += 1;
            }
        }
    }
    return counter;
};

File: 1925-count-square-sum-triples/1925-count-square-sum-triples.java
class Solution {
    public int countTriples(int n) {
        //         counter = 0
        // for i in range(1,n+1,1):
        //     for j in range(1,n+1,1):
        //             if sqrt(i*i + j*j).is_integer() and  sqrt(i*i + j*j) <=n:
        //                 counter +=1
        // return counter 
        int counter = 0;
        for(int i = n; i >= 1; i--){
            for(int j = n; j>= 1; j--){
                double c = Math.sqrt(i*i + j*j);
                if (c == (int)c && c <=n ){
                    counter+=1;
                }
            }
        }
        return counter;
    }
}